src/latex/songs2tex.py

```diff
@@ -0,0 +1,76 @@
+import os
```

Comment:
Please describe in a comment above, what the code is doing.

Reply:
Done, check English understanding :p